procedures/katello/branch.md.erb

```diff
@@ -3,25 +3,40 @@
 ## Release Owner
 
 - [ ] Announce upcoming branching to discourse development category a month before
-- [ ] Start extracting i18n strings and pushing the changes into master git branches so Transifex can pick it up
 - [ ] Update Katello Transifex translations:
   - [ ] Create a Transifex account and join the Foreman team
   - [ ] Spin up a Foreman and Katello installation
-  - [ ] Install Transifex client `pip3 install transifex-client --user`
-  - [ ] [Configure the Transifex client](https://projects.theforeman.org/projects/foreman/wiki/Translating#How-to-pull-translations)
+  - [ ] Configure `~/.transifexrc` [as outlined here](https://projects.theforeman.org/projects/foreman/wiki/Translating#How-to-pull-translations)
```

Comment:
It's really about configuring the whole client. The old text was IMHO better.
```suggestion
  - [ ] [Configure the Transifex client](https://projects.theforeman.org/projects/foreman/wiki/Translating#How-to-pull-translations)
```

Reply:
I had changed this so that the next person who attempts this won't have the same problems I had. I will update it but we'll have to keep an eye on it and see if it was just me..